rfcs/README.md

```diff
@@ -2,14 +2,58 @@
 
 We're trying a new approach of checking in Fluent UI RFCs to the repo so that they're more persistent, easier to find, and we can use PR comments to discuss the proposals.
 
+## Why RFCs?
+
+The RFC process is intended to bring a few benefits:
+
+- More efficent and durable decision making
+  - **Efficent** as we can all engage and review on our own time, rather than locking X engineers in a room for hours
+  - **Durable** as we give more time for decisions to be reviewed, commented, and considered. Rather than making a decision
+    after being locked in a room for an hour with X other engineers.
+- A more inclusive review proces
+  - Supporting both quick and deep thinkers
+  - Allowing proposals and comments from contributors
+- Capturing decisions, allowing:
+  - Us to review recent decisions and build ontop of them
+  - Others to join the project more easily
+
+The React RFC Process sums it up nicely[1]
+
+> The RFC process is a great opportunity to get more eyeballs on your proposal before it becomes a part of a released version. Quite often, even proposals that seem "obvious" can be significantly improved once a wider group of interested people have a chance to weigh in.
+>
+> The RFC process can also be helpful to encourage discussions about a proposed feature as it is being designed, and incorporate important constraints into the design while it's easier to change, before the design has been fully implemented.
+
 ## RFC process
 
-### Creation
+### 1. Creation
+
+- Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+- Write the RFC
+- Submit a PR and label it with **Type: RFC**.
+
+### 2. Review
 
-Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+A key purpose of the review phase is to seek feedback to help improve your idea. You the author are ultimately accountable for the proposal and it's impact. Reviewers can, through comments, help you make the best decision you can. Throughout this review and process it is important that we strive to avoid both [Design by Committee](https://en.wikipedia.org/wiki/Design_by_committee) and [Diffusion of Responsibility](https://en.wikipedia.org/wiki/Diffusion_of_responsibility).
```

Comment:
```suggestion
A key purpose of the review phase is to seek feedback to help improve your idea. You the author are ultimately accountable for the proposal and its impact. Reviewers can, through comments, help you make the best decision you can. Throughout this review and process it is important that we strive to avoid both [Design by Committee](https://en.wikipedia.org/wiki/Design_by_committee) and [Diffusion of Responsibility](https://en.wikipedia.org/wiki/Diffusion_of_responsibility).
```